applications/MeshingApplication/custom_utilities/mmg/mmg_utilities.h

```diff
@@ -789,6 +789,7 @@ class KRATOS_API(MESHING_APPLICATION) MmgUtilities
     SizeType mEchoLevel = 0;                                               /// The echo level of the utilities
     bool mRemoveRegions = false;                                           /// Cuttig-out specified regions during surface remeshing
     DiscretizationOption mDiscretization = DiscretizationOption::STANDARD; /// Discretization The discretization type
+    bool mUsingMetricTensor = true;                                        /// Controls if the metric used is a tensor (true) or a scalar (false)
```

Comment:
Use a flag instead?

Reply:
Do you mean a flag that the user can input, or just use a Flag instead of bool?

Note that the variable `mUsingMetricTensor` is currently automatically assigned, depending on what values are loaded in the modelpart:
https://github.com/KratosMultiphysics/Kratos/blob/f457525e237aeca0e9f7895e05b77fc60290f7af/applications/MeshingApplication/custom_utilities/mmg/mmg_utilities.cpp#L4214